### PR TITLE
Implemented command to download the Hermes Sampling Profiler to a local machine

### LINKED ISSUE #1: Add a command to download Hermes profile
### Add a command that pulls the latest Hermes profile from the Android device to the local machine
Proposals: 
- [x] `react-native profile-hermes dstfilepath`: pull the latest profile generated by the Android device to the local machine
- [x] `react-native profile-hermes dstfilepath --filename [string]`: pull a specific profile generated by the Android device to the local machine

### PR DESCRIPTION
**Pull request by jessieAnhNguyen**
## Summary
- Solves #1 
- Add ```packages/cli/src/commands/profile```: a folder for Hermes Sampling Profile related commands
- File ```packages/cli/src/commands/profile/index.ts```: exporting the default command ```profile-hermes [destinationDir] --fileName [string]```
- File ```packages/cli/src/commands/profile/downloadProfile.ts```: implement the functions that help with the above command 👆 

- Command added in the file ```packages/cli/src/commands/index.ts```: add the profile command in the list of ```projectCommands```

## Test Plan:
- I first followed this guide on how to test changes [Testing changes](https://github.com/MLH-Fellowship/cli/blob/master/CONTRIBUTING.md#testing-your-changes).
- After that, the usage of the command is:
   - Navigate to the directory of a React Native project 
   -  Run the command ```profile-hermes [destinationDir] --fileName [string]``` with appropriate inputs
       - ```destinationDir```: is optional, provided it with the directory where users want to pull the profile to (if not present, pull to the current React Native root directory)
      - ```--fileName [string]``` an option to specify which profile users want to pull (if not present, pull the latest profile)

- Make sure your emulator is running and adb is connected

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
 {
-  "editor.rulers": [
-    80
-  ],
+  "editor.rulers": [80],
```
Comment (jevakallio) [inline .vscode/settings.json]: Can we undo this change? It's probably prettier, but this might make it harder to merge if someone else has edited this file?

**Review by jevakallio — APPROVED** [review]
LGTM :shipit: 😍

**Review by saphal1998 — APPROVED** [review]
LGTM!
